### ISSUE TITLE
[macOS] Project Manager: "Show logs" button does nothing

### ISSUE THREAD

**Issue by LB-KacperKapusta** [o3de/o3de#8228]
**Describe the bug**
Clicking on "Show logs" button available during project building inside the O3DE Project Manager does seemingly nothing.

Please refer to the attached video for more information. 

**Prerequisites**
- O3DE Project Manager is built using macOS SDK layout.

**Steps to reproduce**
1. Open Project Manager.
2. Create a new project.
3. Start building the created project found under My Projects tab.
4. Press "Show logs" button (available once the building starts).

**Expected behavior**
A window displaying building log opens.

**Actual behavior**
Nothing happens. Click is ignored.

**Video**

https://user-images.githubusercontent.com/86953659/157643105-28ef6e75-68bf-47c1-8e1c-ef4976f6b9a9.mp4

**Found in Branch**
Development (7a75e34)

**Desktop/Device**
- Device: MacBook Pro
- OS: macOS
- Version: Big Sur 11.3.1
- CPU: Intel Core i5 2.0GHz Quad-Core
- GPU: Intel Iris Plus Graphics 1536 MB
- Memory: 16GB

**Comment by LB-KacperKapusta** [comment]
This issue has been logged due to a request in the comment section of https://github.com/o3de/o3de/issues/5278.

**Comment by monroegm** [comment]
I believe this has been fixed since it was reported

**Comment by LB-KacperKapusta** [comment]
During verification of the issue on Development (8a9db6c) we encountered another one with projects not appearing in the Project Manager after being created which makes the "Show logs" button not accessible:

Prerequisites: O3DE Project Manager is built using macOS SDK layout.
1. Open Project Manager.
2. Create a new project.

The created project does not appear in the Projects tab. The same happens when attempting to use "Open a project" option.
Should this be logged as a separate issue?

https://user-images.githubusercontent.com/86953659/198262047-d8023379-2d95-461a-a850-2480d7f92c8f.mov

**Comment by LB-KacperKapusta** [comment]
It has been observed that the issue mentioned in the comment above, due to the macOS default project path, is https://github.com/o3de/o3de/issues/3682. Sorry for posting an irrelevant comment.

It has been reverified creating a project in a path outside of the root directory that the "Show logs" button still does nothing during building.
Reopening the issue as it still occurs.

https://user-images.githubusercontent.com/86953659/198568504-86432241-fa28-462a-b251-f54b388b65f9.mov
